src/app/header/header.component.ts

```diff
@@ -1,13 +1,17 @@
 import { Component, OnInit } from '@angular/core';
+import {TranslateService} from '@ngx-translate/core';
 
 @Component({
   selector: 'ia-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
-
-  constructor() { }
+  public selected = 'ru';
```

Comment:
Now this setting will be reset every time the page is refreshed.
As a separate ticket - save selected language in a local storage and get it from there instead of using hardcoded value. Create a localStorage.service to incapsulate working with storage in it.